code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+		to_chat(H, "<span class='userdanger'/span> The Justiciar requires you to have your helmet to unlock your potential.")
+		return FALSE
+	if(!istype(H.shoes, /obj/item/clothing/shoes/combat/swat/bronze))
+		to_chat(H, "<span class='userdanger'/span> The clockwork Justiciar requires you to wear his treads.")
+		return FALSE
+	if(!istype(H.gloves, /obj/item/clothing/gloves/combat/clockwork))
+		to_chat(H, "<span class='userdanger'/span> Brass gauntlets are essential to enable your power.")
+		return FALSE
+	if(!istype(H.glasses, /obj/item/clothing/glasses/hud/clock))
+		to_chat(H, "<span class='userdanger'/span> Unlock your True Sight through my visor before activating this.")
+		return FALSE
+	affecting = H
+	ADD_TRAIT(src, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_helmet = H.head
+	ADD_TRAIT(c_helmet, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_shoes = H.shoes
+	ADD_TRAIT(c_shoes, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_gauntlets = H.gloves
+	ADD_TRAIT(c_gauntlets, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_visor = H.glasses
+	ADD_TRAIT(c_visor, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	return TRUE
+
+/obj/item/clothing/suit/space/clockwork/proc/unlock_suit()
+	affecting = null
+	REMOVE_TRAIT(src, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_helmet)//Should be attached, might not be attached.
+		REMOVE_TRAIT(c_helmet, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_shoes)
+		REMOVE_TRAIT(c_shoes, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_gauntlets)
+		REMOVE_TRAIT(c_gauntlets, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+
+// Initialize messages
+
+#define CLOCK_LOCK_PHASE 				1
+#define CLOCK_COMPLETE_PHASE	 		6
+#define CLOCK_DEINIT_LOGOFF_PHASE		1
+
+GLOBAL_LIST_INIT(clock_initialize_messages, list(
+	"Redirecting the remains of the Justiciar...",
+	"Securing hardsuit...\nConnection established.",
+	"Finding Reebee...",
+	"Connection with Reebee failed.",
+	"Siphoning remains of power into suit...",
+	"Ratvar's last efforts located.",
+	"All systems operational, "
+))
+
+GLOBAL_LIST_INIT(clock_deinitialize_messages, list(
+	"Now de-initializing...",
+	"Shutting down connection with the <B>Justiciar</B>.",
+	"Using the Justiciar's spare power...",
+	"Repelling Repulsive Nar'Sian influences...",
+	"Purifying suit from unholy influences.",
+	"Purification complete.",
+	"Unsecuring hardsuit mechanism."
+))
+
+
+/obj/item/clothing/suit/space/clockwork/proc/toggle_on_off()
+	. = TRUE
+	if(s_busy)
+		to_chat(loc, "<span class='warning'/span> The Justiciar's power must wait.")
+		return FALSE
+	s_busy = TRUE
+	if(s_initialized)
+		deinitialize()
+	else
+		cinitialize()
+
+/obj/item/clothing/suit/space/clockwork/ui_action_click(mob/user, action)
+	if(istype(action, /datum/action/item_action/initialize_clock_suit))
+		toggle_on_off()
+		return TRUE
+
+/obj/item/clothing/suit/space/clockwork/proc/cinitialize(delay = s_delay, mob/living/carbon/human/U = loc, phase = 0)
+	if(!U || !U.mind)
+		s_busy = FALSE
+		return
+	if (phase > CLOCK_LOCK_PHASE && (U.stat == DEAD || U.health <= 0))
```

Comment:
```suggestion
	if (phase > CLOCK_LOCK_PHASE && (U.stat == DEAD))
```
there are multiple ways to retain the ability to act while at 0 health or less (most notably, the usage of penthrite); it'd kind of suck for this suit to just pop off of you mid-combat just because it thinks you've been incapacitated when you actually haven't